woodwork/table_accessor.py

```diff
@@ -1124,6 +1124,19 @@ def infer_temporal_frequencies(self, temporal_columns=None):
             self._dataframe, temporal_columns=temporal_columns
         )
 
+    @_check_table_schema
+    def validate(self, return_indices=False):
+        if return_indices:
+            indices = [
```

Comment:
Should this be the results for all columns or only the columns that contain invalid values?

Reply:
I think it should be results for all columns to keep data out of memory for dask and koalas